packages/mobile/src/geth/actions.ts

```diff
@@ -15,6 +17,20 @@ export const setInitState = (state: InitializationState): SetInitState => ({
   state,
 })
 
+export const cancelGethSaga = () => ({
+  type: Actions.CANCEL_GETH_SAGA,
+})
+
+interface SetPromptZeroSync {
```

Comment:
Nit, I think the name convention we use elsewhere is `SetPromptZeroSyncAction`, though I guess not in this file for some reason

Reply:
Updated all actions in this file to use the normal naming convention